package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "subscribe-ui-event",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

Comment:
BTW, internally our ci tool does the tags stuff for us automatically, so we don't handle that, but open source we did.